executor/src/witgen/jit/compiler.rs

```diff
@@ -48,10 +52,26 @@ impl<T: FieldElement> WitgenFunction<T> {
             params: params.into(),
             mutable_state: mutable_state as *const _ as *const c_void,
             call_machine: call_machine::<T, Q>,
+            fixed_data: fixed_data as *const _ as *const c_void,
+            get_fixed_value: get_fixed_value::<T>,
         });
     }
 }
 
+extern "C" fn get_fixed_value<T: FieldElement>(
+    fixed_data: *const c_void,
+    column: u64,
+    row: u64,
+) -> T {
+    let fixed_data = unsafe { &*(fixed_data as *const FixedData<'_, T>) };
+    let poly_id = PolyID {
+        id: column,
+        ptype: PolynomialType::Constant,
+    };
+    // TODO which size?
```

Comment:
Does it always work to use the largest one?

Reply:
But that's not the issue - the question is: Can we pick the largest size and assume it's the same value in the other sizes?

Reply:
That's the assumption we currently go with already ;) We could assert that as well.